tests/base/src/test/java/org/wildfly/security/auth/FileSystemSecurityRealmTest.java

```diff
@@ -79,6 +86,22 @@
 // has dependency on wildfly-elytron-realm, wildfly-elytron-auth-server, wildfly-elytron-credential
 public class FileSystemSecurityRealmTest {
 
+    private static final Provider provider = WildFlyElytronPasswordProvider.getInstance();
+
+    public FileSystemSecurityRealmTest() throws GeneralSecurityException {
+    }
+
+    @BeforeClass
+    public static void onBefore() throws Exception {
+        Security.addProvider(provider);
+    }
+
+    @AfterClass
+    public static void onAfter() throws Exception {
+        Security.removeProvider(provider.getName());
+    }
+    SecretKey key = SecretKeyUtil.generateSecretKey(128);
```

Comment:
Could make this private static final

Reply:
It can't be static due to the `GeneralSecurityException` it throws, changed to `private final` instead